packages/api-entrypoint/package.json

```diff
@@ -1,43 +1,24 @@
 {
-  "name": "@pooltogether/api-entrypoint",
+  "private": true,
+  "name": "api-yield-source-data",
```

Comment:
What was wrong with `@pooltogether/api-entrypoint` as a name? Shouldn't we also rename the `packages/api-entrypoint` folder if we're going to rename the project?